operator/validator/controller.go

```diff
@@ -928,6 +928,8 @@ func (c *controller) onShareInit(share *ssvtypes.SSVShare) (*validators.Validato
 		opts.SSVShare = share
 		opts.Operator = operator
 		opts.DutyRunners = SetupRunners(validatorCtx, c.logger, opts)
+		opts.ValidatorIndex = share.ValidatorIndex
```

Comment:
Why do we need to pass it if we already pass `SSVShare` and can get `ValidatorIndex` from there? If there's a reason for doing this, can you please leave a comment on why?

Reply:
you're right, using share instead.